src/google-classroom-extractor/edfi_google_classroom_extractor/mapping/assignment_submissions.py

```diff
@@ -4,10 +4,63 @@
 # See the LICENSE and NOTICES files in the project root for more information.
 
 from typing import Dict, Tuple, Any
-from pandas import DataFrame
+from pandas import DataFrame, isna
 from edfi_google_classroom_extractor.mapping.constants import SOURCE_SYSTEM
 
+# States returned by API
+CREATED_STATE = "CREATED"
+NEW_STATE = "NEW"
+RECLAIMED_STATE = "RECLAIMED_BY_STUDENT"
 TURNED_IN_STATE = "TURNED_IN"
+RETURNED_STATE = "RETURNED"
```

Comment:
```suggestion
class API_STATES:
  CREATED_STATE = "CREATED"
  NEW_STATE = "NEW"
  RECLAIMED_STATE = "RECLAIMED_BY_STUDENT"
  TURNED_IN_STATE = "TURNED_IN"
  RETURNED_STATE = "RETURNED"
```
This can improve readability

Reply:
Not sure it does. I'd think "API_STATES.NEW_STATE" would have redundant naming, so I'd want to drop the "_STATE" suffix, making it "API_STATES.NEW". Not much different, but longer than "NEW_STATE" without a readablility improvement. And on other modules I might want to import "from API_STATES import NEW", and now I'm using "NEW" where "NEW_STATE" would be clearer. Could use the shorter class name "STATE", but same issue with above import, while a plain "import STATE" results in "STATE.NEW". Not really a readability difference between "NEW_STATE" and "STATE.NEW".